pkg/tfbridge/x/token_test.go

```diff
@@ -587,10 +587,10 @@ func TestMaxItemsOneAliasing(t *testing.T) {
                 "current": "pkg:index/r1:R1",
                 "fields": {
                     "f1": {
-                        "maxItemOne": false
+                        "maxItemsOne": false
```

Comment:
So this got renamed?

Reply:
Yes. See https://github.com/pulumi/pulumi-terraform-bridge/pull/1027#discussion_r1175653263.